### PR TITLE
GrapesJs as default builder

### PR DESCRIPTION
**Pull request by kuzmany**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for 4.0
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | yes
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 

#### Description:
This make GrapesJs as default builder during the installation.

#### Steps to test this PR:
1. Make new installation
2. Go and open email
3. GrapesJs should be default builder

### CODE REVIEW

File: plugins/GrapesJsBuilderBundle/Config/config.php
Comment (RCheesley) [inline plugins/GrapesJsBuilderBundle/Config/config.php]:
```suggestion
    'author'      => 'Mautic Community',
```

File: plugins/GrapesJsBuilderBundle/Config/config.php
Comment (RCheesley) [inline plugins/GrapesJsBuilderBundle/Config/config.php:8]:
```suggestion
```
Since the composer work we get the version from [Packagist](https://packagist.org/packages/mautic/grapes-js-builder-bundle) - I think therefore that we should not be explicitly stating it in this way

**Review by RCheesley — CHANGES_REQUESTED** [review]
PR works as described but it requires test coverage before we can merge - are you able to do that @kuzmany ?

Also a couple of small suggestions for the config file.

File: plugins/GrapesJsBuilderBundle/Config/config.php
Comment (kuzmany) [reply plugins/GrapesJsBuilderBundle/Config/config.php:8]: Are we sure the plugin bundle do not need version? If we need made process some migration during plugin update? I think yes.

During the `mautic:plugins:reload` Mautic check plugin version and If version was updated then execute some code to run plugin migration etc...

Comment (RCheesley) [reply plugins/GrapesJsBuilderBundle/Config/config.php:8]: I did check that with @nickveenhof  but let's double check!

Comment (kuzmany) [reply plugins/GrapesJsBuilderBundle/Config/config.php:8]: I am talking about this part - https://developer.mautic.org/#plugin-schema
Without plugin versioning, it's not possible do it.

**Comment by tejasync** [comment]
@kuzmany 
There is some scope for improvement and added test cases for codecov.
You could find it in here: https://github.com/kuzmany/mautic/pull/18

**Comment by tejasync** [comment]
@kuzmany The CS Fixer failed.
I have fixed it in here https://github.com/kuzmany/mautic/pull/19.

**Comment by tejasync** [comment]
@RCheesley This PR works as expected and is ready-to-commit! Although, I had made few basic changes to the code, could you review, please?

**Comment by RCheesley** [comment]
Awesome! I guess that with the tests that @ts-navghane has added then we may not need [this PR](https://github.com/mautic/mautic/pull/10006) any more? Or can that work be incorporated here?

**Review by RCheesley — APPROVED** [review]
This works great - we have a bug with the builder at present which is preventing me from testing the actual builder interface, however the functionality works as expected (builder enabled by default, able to disable at will) and I have tested the PR before so LGTM! 🚀

**Comment by RCheesley** [comment]
@all-contributors please add @ts-navghane for test, code, review and userTesting

**Comment by RCheesley** [comment]
@all-contributors please add @ts-navghane for review

File: app/bundles/InstallBundle/Tests/InstallFixtures/ORM/GrapeJsDataTest.php
Comment (escopecz) [inline app/bundles/InstallBundle/Tests/InstallFixtures/ORM/GrapeJsDataTest.php:57]: Why is the version important for this test? It will have to be updated every time a new version of the plugin is released, right? Or if the description will change. I'd leave only the bundle name in the test.

Comment (RCheesley) [reply app/bundles/InstallBundle/Tests/InstallFixtures/ORM/GrapeJsDataTest.php:57]: Agree that we should not be setting the version anywhere (it's now set in the composer file for plugins and themes).  Must have been missed in the code review :(

**Pull request merged**
